app/src/main/java/org/mozilla/focus/experiments/ExperimentalTabsFeatures.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.experiments
+
+import org.mozilla.experiments.nimbus.Variables
+
+private const val VARIABLE_TAB_MODE = "mode"
+
+private const val TAB_MODE_MULTI = "multi"
+private const val TAB_MODE_SINGLE = "single"
+
+/**
+ * Experimental features regarding the usage of "browser tabs".
+ */
+class ExperimentalTabsFeatures(
+    variables: Variables
+) {
+    /**
+     * Do we support multiple tabs or is this a single tab browser?
+     */
+    val isMultiTab: Boolean by lazy {
+        when (variables.getString(VARIABLE_TAB_MODE)) {
```

Comment:
The feature API supports boolean types, if you wanted to you could simplify this a bit with something like `variables.getBool(IS_MULTI_TAB)`

Reply:
Thanks! I updated the patch to use a boolean instead.